modules/m_services.c

```diff
@@ -245,6 +246,11 @@ me_rsfnc(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source
 			use_id(target_p), parv[2], (long) target_p->tsinfo);
 
 	del_from_client_hash(target_p->name, target_p);
+
+	/* invalidate nick delay because we're forcing this nick to be used */
+	if((nd = rb_dictionary_retrieve(nd_dict, parv[2])))
```

Comment:
```suggestion
	nd = rb_dictionary_retrieve(nd_dict, parv[2]);
	if (nd != NULL)
```

No need to use the value of an assignment here, IMO.  
<sub><sub>(and no space after a keyword is icky)</sub></sub>